compiler/qsc_frontend/src/resolve.rs

```diff
@@ -68,6 +68,10 @@ pub(super) enum Error {
     #[error("`{0}` not found")]
     #[diagnostic(code("Qsc.Resolve.NotFound"))]
     NotFound(String, #[label] Span),
+
+    #[error("This name results in ambiguity in a prelude namespace.")]
+    #[diagnostic(code("Qsc.Resolve.PreludeAmbiguity"))]
+    PreludeAmbiguity(#[label] Span),
```

Comment:
I'd keep the naming close to the other error about ambiguous names. So maybe `Ambiguous` and `AmbiguousPrelude`?